Dockerfile

```diff
@@ -46,5 +49,5 @@ RUN ln -sf /var/lib/rabbitmq/.erlang.cookie /root/
 COPY docker-entrypoint.sh /
 ENTRYPOINT ["/docker-entrypoint.sh"]
 
-EXPOSE 5672 4369 25672
+EXPOSE 4369 5671 5672 25672
```

Comment:
Sorry, looks like #52 caused a merge conflict here. :disappointed:  I like the numeric ordering you've got here better though! :+1: